codegen/cmd/injection-gen/generators/informer.go

```diff
@@ -124,4 +175,49 @@ func Get(ctx {{.contextContext|raw}}) {{.informersTypedInformer|raw}} {
 	}
 	return untyped.({{.informersTypedInformer|raw}})
 }
+
+type wrapper struct {
+	client    {{.clientSetInterface|raw}}
```

Comment:
```suggestion
	client    {{ .clientSetInterface|raw }}
```

Reply:
nit, half the file does that way and half does not add spaces, so idk, I don't care too much.

Reply:
Yeah, the squished way was making my eyes bleed.  Perhaps we should follow up with a formatting pass?